### DISCUSSION TITLE
nuclei fuzzing seems broken on all 3.2.x

### DISCUSSION THREAD

**Discussion by bpavesi** [projectdiscovery/nuclei#5109] [Q&A] [answered]
### Nuclei version:
3.2.5 to 3.2.0

### Current Behavior:
I would try to fuzz a path with nuclei new fuzzing features and i noticed that a fuzzing template of type query that work on 3.1.6 is not working from 3.2.0 to 3.2.5. Also path fuzzing seems broken.

In all test nuclei end with: [FTL] Could not run nuclei: no templates provided for scan

### Expected Behavior:
fuzzing template of type query that work on 3.1.6 should work also on 3.2.5

**Comment by geeknik** [comment, marked as answer]
I believe you need to use the `-dast` flag since `-fuzz` is now deprecated. Let me know if that doesn't work for you.

**Comment by bpavesi** [comment]
@geeknik 
with -dast flag it works! 

tnx
